q2_feature_table/_summarize/tabulate_seqs_assets/index.html

```diff
@@ -1,31 +1,90 @@
 {% extends "base.html" %}
 
 {% block content %}
+  <div class="row">
+    <div class="col-lg-6">
+      <h3>Sequence Length Statistics</h3>
+      <table class="table table-striped" id="stats_table">
+        <thead>
+          <tr>
+            <th>Sequence Count</th>
+            <th>Min Length</th>
+            <th>Max Length</th>
+            <th>Mean Length</th>
+            <th>Range</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>{{ stats.count }}</td>
+            <td>{{ stats.min }}</td>
+            <td>{{ stats.max }}</td>
+            <td>{{ stats.mean | round(2) }}</td>
+            <td>{{ stats.range }}</td>
+          </tr>
+        </tbody>
+      </table>
+    </div>
+    <div class="col-lg-6">
+      <h3>Seven-Number Summary of Sequence Lengths</h3>
+      <table class="table table-striped" id="summary_table">
+        <thead>
+          <tr>
+            <th>Percentile:</th>
+            {% for percentile in stats.seven_num_summ_percentiles %}
+            <th>{{ percentile | int }}%</th>
+            {% endfor %}
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+          <td>Length (nts):</td>
+          {% for value in stats.seven_num_summ_values %}
+            <td>{{ value | round(2) }}</td>
```

Comment:
Can you round these values down, and add a short note somewhere saying the floor of the values was taken.

Reply:
Don't worry about rounding in your helper function, just in the template rendering is fine.